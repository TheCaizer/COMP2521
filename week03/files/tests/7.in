17
9
5
3
2
4
7
6
8
13
11
10
12
15
14
16