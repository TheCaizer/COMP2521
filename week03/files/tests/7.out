Tree:
               17
               /
              9
             / \
            /   \
           /     \
          /       \
         /         \
        /           \
       /             \
      5              13
     / \             / \
    /   \           /   \
   /     \         /     \
  3       7       /       \
 / \     / \     11       15
2   4   6   8   / \       / \
               /   \     /   \
              10   12   14   16

#nodes = 16,  #leaves = 8
Infix  : 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 
Prefix : 17 9 5 3 2 4 7 6 8 13 11 10 12 15 14 16 
Postfix: 2 4 3 6 8 7 5 10 12 11 14 16 15 13 9 17 
ByLevel: 17 9 5 13 3 7 11 15 2 4 6 8 10 12 14 16 
